specs/latest/2.0/index.html

```diff
@@ -2034,6 +2034,33 @@ <h2 class="no-toc">Table of contents</h2>
 
           <p>Returns null if any OpenGL errors are generated during the execution of this
           function.</p>
+
+          <p>In order to ensure consistent behavior across platforms, queries' results must only be made
+          available when the user agent's <a
+          href="http://www.whatwg.org/specs/web-apps/current-work/multipage/webappapis.html#event-loops">event
+          loop</a> is not executing a task. In other words:
+          <ul>
+            <li> A query's result must not be made available until the current frame has completed
```

Comment:
This is not a rephrasing of the above, but rather a more strict restriction. Requiring returning to the event loop is different than requiring that the current frame is complete.

I hesitantly support the former, but do not support the latter.